### ISSUE TITLE
German and French Localization not working on Enchanting

### ISSUE THREAD

**Issue by scholaly** [Chalos-Atiesh/GuildTradeskills#71]
See Profession.lua Line 146:
```
	if profession.professionName == 'Enchanting' then
		GetNumTradeSkills = GetNumCrafts
		GetTradeSkillItemLink = GetCraftItemLink
		GetTradeSkillNumReagents = GetCraftNumReagents
		GetTradeSkillReagentInfo = GetCraftReagentInfo
		GetTradeSkillReagentItemLink = GetCraftReagentItemLink

		GetTradeSkillInfo = function(i)
			local name, _, kind, num = GetCraftInfo(i)
			return name, kind, num
		end
	end
```

Hard coded String 'Enchanting' will never be found on german or french client.

**Comment by Chalos-Atiesh** [comment]
Whoops. That should be localized. I'll have time to get it out in a day or so.

**Comment by Chalos-Atiesh** [comment]
This should be fixed with 6.0.5: https://github.com/Chalos-Atiesh/GuildTradeskills/commit/d184860d72a60fcb16cda5987c40ce6236cf730c

**Comment by scholaly** [comment]
It's working fine now. Thanks =)
